research/flamby/fed_isic2019/fenda/run_fold_experiment.slrm

```diff
@@ -138,6 +139,7 @@ do
             --client_number ${c} \
             --learning_rate ${CLIENT_LR} \
             --server_address ${SERVER_ADDRESS} \
+            --type_run cos_sim \
```

Comment:
Does batch properly interpret `cos_sim` as a string here without the quotes? I feel like maybe it doesn't?

Reply:
In my runs there were no issues, as I was checking whether they were running with true losses or not.